### PR TITLE
Remove wide strings where possible and switch to format

### LINKED ISSUE #888: Remove wide strings, use format
Wide strings were mainly added for the old UI system - now that we're using ImGui we can remove most of these and stop writing L everywhere.

We can also use format now instead of stringstream - in the cases that happens.

### PR DESCRIPTION
**Pull request by chreden**
Remove wide strings where possible.
Switch to std::format instead of adding std::to_string together or stringstreams.
Closes #888

### CODE REVIEW

File: trlevel/trtypes.h [hunk 1]
```diff
@@ -17,6 +17,7 @@ namespace trlevel
         FogBulb = 4
     };
 
+    constexpr float Scale{ 1024.0f };
```
Comment (makotocchi) [inline trlevel/trtypes.h:20]: Missing a space

File: trview.app/Elements/ITrigger.cpp [hunk 2]
```diff
@@ -4,61 +4,61 @@ namespace trview
...
+            { TriggerType::Antipad, "Antipad" },
+            { TriggerType::Combat, "Combat" },
+            { TriggerType::Dummy, "Dummy" },
+            { TriggerType::AntiTrigger, "Antitrigger" },
+            { TriggerType::HeavySwitch, "Heavy Switch" },
+            { TriggerType::HeavyAntiTrigger, "Heavy Antitrigger" },
+            { TriggerType::Monkey, "Monkey" },
+            { TriggerType::Skeleton, "Skeleton" },
+            { TriggerType::Tightrope, "Tightrope" },
+            { TriggerType::Crawl, "Craw" },
```
Comment (makotocchi) [inline trview.app/Elements/ITrigger.cpp:24]: Missing an L

**Review by makotocchi — CHANGES_REQUESTED** [review]

File: trview.app/Elements/ITrigger.cpp (hunk 2 above)
Comment (makotocchi) [reply trview.app/Elements/ITrigger.cpp:24]: Line 24, Github decided to highlight the other lines too.

File: trlevel/trtypes.h (hunk 1 above)
Comment (chreden) [reply trlevel/trtypes.h]: Added the space

File: trview.app/Elements/ITrigger.cpp (hunk 2 above)
Comment (chreden) [reply trview.app/Elements/ITrigger.cpp]: Readded the l

**Review by makotocchi — APPROVED** [review]

**Pull request merged**